app/src/main/java/com/odysee/app/model/Claim.java

```diff
@@ -444,13 +445,17 @@ public static Claim fromJSONObject(JSONObject claimObject) {
      * @param claimId - The claimId to used for the claim
      * @param liveUrl - The url which should be used for connecting to the livestream
      * @param livestreamViewers - The amount of viewers currently watching the stream
+     * @param startTime - The start time of the stream in ISO 8601 UTC format
      * @return A new Claim object
      */
-    public static Claim fromLiveStatus(String claimId, String liveUrl, int livestreamViewers) {
+    public static Claim fromLiveStatus(String claimId, String liveUrl, int livestreamViewers, String startTime) {
         Claim claim = new Claim();
         claim.setClaimId(claimId);
         claim.setLivestreamUrl(liveUrl);
         claim.setLivestreamViewers(livestreamViewers);
+        StreamMetadata value = new StreamMetadata();
+        value.setReleaseTime(ZonedDateTime.parse(startTime).toInstant().getEpochSecond());
```

Comment:
Catch DateTimeParseException here and everywhere you used parse(). printStackTrace the exception. Return null